discord/hook.go

```diff
@@ -0,0 +1,81 @@
+package discord
+
+import (
+	"os"
+	"time"
+
+	"github.com/kz/discordrus"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	webhookURLEnvName = "DISCORD_WEBHOOK_URL"
+)
+
+// Hook is a Discord logger hook.
+type Hook struct {
+	logrus.Hook
+	limit      time.Duration
+	timestamps map[string]time.Time
+}
+
+// Option defines an option for Discord logger hook.
+type Option func(*Hook)
+
+// WithLimit sets enables logger rate limiter with specified limit.
```

Comment:
It should be just `enables`, not `sets`, right?

Reply:
@Darkren Yes, thanks